test/ussd_health_worker.test.js

```diff
@@ -645,6 +646,43 @@ describe("familyconnect health worker app", function() {
                     .run();
             });
         });
+
+        describe("utils function testing", function() {
+            describe("is_valid_name", function() {
+                it('should return a true/false depending on validity of name', function() {
+                    // test data
+                    var testDataArray = [
+                        'john',
+                        'Sally',
+                        'Mark Anthony',
+                        ' Peter',
+                        'J0hn',
+                        'Jan-Hendrik',
+                        '12345',
+                        'Zoë',
+                        'Al-YoãHukáé'
+                    ];
+
+                    // function call
+                    var resultsArray = [];
+                    for (var i=0; i<testDataArray.length; i++) {
+                        resultsArray.push(go.utils.is_valid_name(testDataArray[i], 0, 150));
+                    }
+
+                    // expected results
+                    assert.equal(resultsArray.length, 9);
+                    assert.equal(resultsArray[0], true);
+                    assert.equal(resultsArray[1], true);
+                    assert.equal(resultsArray[2], true);
+                    assert.equal(resultsArray[3], true);
+                    assert.equal(resultsArray[4], false);
```

Comment:
Please add a comment why it's false (eg. number in J0hn rejected), and add a test for one of the other punctuation marks that should fail.